index.html

```diff
@@ -97,5 +97,214 @@ <h2 class="more_det">LET’S CONNECT</h2>
+        have a project you need coded, don’t<br />
+        hestiate to contact me.
+      </p>
+      <div class="media_div2">
+        <h4 class="more_det1">LET’S CONNECT</h4>
+      </div>
+      <ul class="media_u">
+        <li>
+          <a
+            class="github2"
+            href="https://github.com/karam084"
+            target="_blank"
+            rel="noopener noreferrer"
+            ><img src="media-icons/github.png" alt="github"
+          /></a>
+        </li>
+        <li>
+          <a
+            class="linked2"
+            href="https://www.linkedin.com/in/karam-fouad-179830214/"
+            target="_blank"
+            rel="noopener noreferrer"
+            ><img src="media-icons/Vector0.png" alt="linked" /><img
+              src="media-icons/link.png"
+              alt="linkedin"
+          /></a>
+        </li>
+        <li>
+          <a class="medium2" href="#" target="_blank" rel="noopener noreferrer"
+            ><img src="media-icons/medium.png" alt="medium"
+          /></a>
+        </li>
+        <li>
+          <a
+            class="twitter2"
+            href="https://twitter.com/ElarabFouad"
+            target="_blank"
+            rel="noopener noreferrer"
+            ><img src="media-icons/twitter@2x.png" alt="twitter"
+          /></a>
+        </li>
+        <li>
+          <a
+            class="angellist2"
+            href="#"
+            target="_blank"
+            rel="noopener noreferrer"
+            ><img src="media-icons/angellist.png" alt="angellist"
+          /></a>
+        </li>
+      </ul>
+      <button class="flex-botu" type="button">Get my resume</button>
+      <div class="fram-work">
+        <ul class="border-lang">
+          <li>
+            Langguages<a href="#" rel="noopener noreferrer" class="vector-1"
+              ><i class="arrow-down"></i
+            ></a>
+          </li>
+        </ul>
+        <ul class="lang-img">
+          <li>
+            <a href="#" class="js-icon"
+              ><img src="src-image/js-icon.png" alt="js-icon" />JavaSript</a
+            >
+          </li>
+          <li>
+            <a href="#" class="html-icon"
+              ><img src="src-image/html-icon.png" alt="html-icon" />HTML</a
+            >
+          </li>
+          <li>
+            <a href="#" class="css-icon"
+              ><img src="src-image/css-icon.png" alt="css-icon" />CSS</a
+            >
+          </li>
+        </ul>
+        <ul class="fr-work">
+          <li class="border-frem">
+            Frameworks<a
+              margin-left:
+              10px;href="#"
+              rel="noopener noreferrer"
+              class="vector-2"
+              ><i class="arrow-right"></i
+            ></a>
+          </li>
+          <li class="border-skill">
+            Skills<a href="#" rel="noopener noreferrer" class="vector-3"
+              ><i class="arrow-right"></i
+            ></a>
+          </li>
+        </ul>
+      </div>
+    </section>
   </body>
```

Comment:
# Optional but Important!
- [ ] [OPTIONAL] There is a horizontal scroll on your portfolio page, this could be as a result of `width: 360px` added to `body` in your CSS file, kindly look at it. also, you might consider adding `overflow: hidden` to `body` in your CSS